spec/integration/thread_spec.rb

```diff
@@ -11,4 +20,34 @@
       expect(subject).to all(be_a(described_class))
     end
   end
+
+  describe '.create' do
+    subject { described_class.create(conversation_id, thread_type, params) }
+    let(:thread_type) { 'notes' }
+    let(:params) do
+      {
+        text: 'Hello, note!'
+      }
+    end
+
+    it 'creates a new thread on a given conversation' do
+      expect(subject.success?).to eq(true)
```

Comment:
This line will change slightly if we update the `Thread.create` method to return `true` on success.